src/EditorFeatures/CSharpTest/Structure/DocumentationCommentStructureTests.cs

```diff
@@ -144,7 +144,7 @@ class Class3
 }";
 
             await VerifyBlockSpansAsync(code,
-                Region("span", "/// <summary> Hello C#!", autoCollapse: true));
+                Region("span", "/// <summary>Hello C#!", autoCollapse: true));
```

Comment:
This change is desirable (IMO).  There is no space between the > and the text in the original code.  So there's no need to add a space here.